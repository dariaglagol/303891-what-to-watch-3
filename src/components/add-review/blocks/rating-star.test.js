import React from "react";
import renderer from "react-test-renderer";
import RatingStar from "./rating-star";

it(`Render star`, () => {
  const starComponent = renderer.create(
      <RatingStar
        item={1}
        onChange={() => {}}
      />
  ).toJSON();

  expect(starComponent).toMatchSnapshot();
});
