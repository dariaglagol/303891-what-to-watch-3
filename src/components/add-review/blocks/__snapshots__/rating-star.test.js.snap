// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render star 1`] = `
Array [
  <input
    className="rating__input"
    id="star-1"
    name="rating"
    onChange={[Function]}
    type="radio"
    value={1}
  />,
  <label
    className="rating__label"
    htmlFor="star-1"
  >
    Rating 
    1
  </label>,
]
`;
